src/app/user/user.service.ts

```diff
@@ -1,8 +1,20 @@
+import { Response, Request } from 'express';
+
+import { IUser } from '../user/user.type';
+
 import User from './user.model';
+
 export class UserService {
+
   private model = User;
 
-  getUser(): string {
-    return '';
+  async getUser(req: Request, res: Response): Promise<void> {
+    try {
+      const user: IUser[] = await this.model.find();
```

Comment:
@hsedaqat1378 please use findOne for this method. based on the name of the method it should find and return a single `User`. if it's gonna return array of users please rename the method to `getUsers` .

Reply:
your mean this right:
this.model.findOne({$where})

Reply:
@hsedaqat1378 yes, but please get rid of $where for the moment